packages/shared/utils/pvc-dropdown.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+import * as _ from 'lodash-es';
+import ResourceDropdown from "../../components/odf-resources/ResourceDropdown";
+import { pvcResource } from "../../models/resources";
+import {
+  PersistentVolumeClaimModel,
+  PersistentVolumeClaimKind
+} from "../selectors/k8s";
+
+export const PVCDropdown: React.FC<PVCDropdownProps> = (props) => {
+  const { onChange } = props;
+  return (
+    <ResourceDropdown
+      className="nb-endpoints-form-entry__dropdown nb-endpoints-form-entry__dropdown--full-width"
+      resource={pvcResource}
+      resourceModel={PersistentVolumeClaimModel}
```

Comment:
create diff folder under `shared` for all dropdowns... similar for other componets as well if necessary...